embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/LogsApi.kt

```diff
@@ -164,12 +174,14 @@ void logCustomStacktrace(@NonNull StackTraceElement[] stacktraceElements,
      * @param isNotification           if it is a notification message.
      * @param hasData                  if the message contains payload data.
      */
-    void logPushNotification(@Nullable String title,
-                             @Nullable String body,
-                             @Nullable String topic,
-                             @Nullable String id,
-                             @Nullable Integer notificationPriority,
-                             @NonNull Integer messageDeliveredPriority,
-                             @NonNull Boolean isNotification,
-                             @NonNull Boolean hasData);
+    fun logPushNotification(
```

Comment:
We should look to deprecate this and replace it with a more normal looking method. The method signature is.... not fantastic.